src/Service.Host/client/src/reducers/reducerFactories/itemStoreFactory.js

```diff
@@ -18,6 +18,12 @@
                     item: null,
                     editStatus: 'create'
                 };
+            case 'REQUEST_SERIAL_NUMBER_COUNTS':
```

Comment:
again would be nice to get this more generic. 
If it does need a specific reducer then it can't be in the generic factory because I've removed it. Sorry.

Reply:
I'll merge in your changes and try to work something out